prjxray-wire-viewer/graph.html

```diff
@@ -0,0 +1,17 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<title>FPGA visualization</title>
+<meta name="description" content="FPGA visualization." />
+<meta charset="UTF-8">
+</head>
+<body>
+<div id="myDiagramDiv" style="width:100%; height:800px; border:1px solid black"></div>
+<script src="go.js"></script>
+<script src="tileconn.js"></script>
+<script src="tilegrid.js"></script>
```

Comment:
The `tileconn.js` and `tilegrid.js` come from the [`prjxray-db`](https://github.com/SymbiFlow/prjxray-db) right? Could you include them via adding a git submodule for prjxray-db in third_party too?